docs/deployment.rst

```diff
@@ -1,15 +1,22 @@
+.. _deployment:
+
 Deployment
 ==========
 
-Serving the UI build from the distribution directory
-----------------------------------------------------
-To serve the UI from the ``dist`` directory, run:
+Serving the UI build from the dist directory
+--------------------------------------------
+This step required prior to deployment concatenates and minifies the js and css and moves all required assets to a directory in the project root called ``dist``. To do this, if you haven't already done so:
+
+* As root, install grunt globally ``npm install grunt-cli -g``
+* Install the ``devDependencies`` in ``package.json`` by running ``npm install`` from the project root
+* Then run the following command in the repo root:
 
 .. code-block:: bash
 
-  (venv)vagrant@precise32:~/treeherder$ grunt build
+    grunt build
```

Comment:
Since grunt isn't available in our default venv even after a `grunt-cli -g` install, I removed the (venv) prompt.